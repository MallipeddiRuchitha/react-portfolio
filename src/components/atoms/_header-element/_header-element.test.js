import React from 'react';
import { shallow ,mount} from 'enzyme';
import HeaderElement from './_header-element';

describe("HeaderElement component",() =>{
    test("Matches the snapshot",() => {
        
        const headerElement = shallow(<HeaderElement />);
        expect(headerElement).toMatchSnapshot();

    })
    test("header element is rendered",() => {
        const wrapper = shallow(<HeaderElement />);
        expect(wrapper.find('#headerElement').exists()).toEqual(true);
        
    });
    

});