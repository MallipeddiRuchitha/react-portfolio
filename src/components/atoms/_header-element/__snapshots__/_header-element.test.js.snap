// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderElement component Matches the snapshot 1`] = `
<WithStyles(ForwardRef(Typography))
  className="makeStyles-styleHeaderElement-1"
  id="headerElement"
  variant="h4"
/>
`;
